commcare_connect/audit/models.py

```
@@ -0,0 +1,66 @@
+    status = models.CharField(max_length=16, choices=Status.choices, default=Status.PENDING)
+    completed_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="completed_audit_reports",
+    )
+    completed_date = models.DateTimeField(null=True, blank=True)
+
+    class Meta:
+        ordering = ["-period_end", "-date_created"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)